### PR TITLE
Remove unused parameter in Deploy.ps1

### PR DESCRIPTION
**Pull request by cegekaJG**
This PR removes the parameter `$token` from "Deploy.ps1", which currently isn't being read by any script, or specified by any workflow.

### CODE REVIEW

File: Actions/Deploy/Deploy.ps1 [hunk 1]
```diff
@@ -1,6 +1,4 @@
 Param(
-    [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
-    [string] $token,
```
Comment (mazhelez) [inline Actions/Deploy/Deploy.ps1:3]: It's not used in the script but it's passed in the action YAML (Actions/Deploy/action.yaml).

Comment (cegekaJG) [reply Actions/Deploy/Deploy.ps1:3]: You're correct, I missed that. But no workflow overrides the default value and the token still goes unused, so shouldn't it go? Or is there something on the roadmap that would require it?

File: Actions/Deploy/action.yaml [hunk 2]
```diff
@@ -5,10 +5,6 @@ inputs:
     description: Shell in which you want to run the action (powershell or pwsh)
     required: false
     default: powershell
-  token:
```
Comment (mazhelez) [inline Actions/Deploy/action.yaml:8]: Lines 32 and 40 need to be cleaned up as well. Also, run the tests in Tests\Deploy.Action.Test.ps1 locally to verify they pass.

File: Actions/Deploy/Deploy.ps1 (hunk 1 above)
Comment (freddydk) [reply Actions/Deploy/Deploy.ps1:3]: It also needs to be removed from the README.md (in the Actions/Deploy folder)

File: Actions/Deploy/action.yaml (hunk 2 above)
Comment (freddydk) [reply Actions/Deploy/action.yaml:8]: @cegekaJG - there is a CI test failing. That test checks whether the .ps1 script matches the .yaml file with parameters etc. once you have removed the token in the lines mentioned by Maria, the test will pass.

Comment (cegekaJG) [reply Actions/Deploy/action.yaml:8]: My mistake, it should work now. The test was successful as well.

**Review by freddydk — CHANGES_REQUESTED** [review]
@cegekaJG please remove `token` from README.md in Actions/Deliver

**Review by freddydk — APPROVED** [review]

**Pull request merged**